extension.js

```diff
@@ -56,14 +57,17 @@ export default class SaneAirplaneMode extends Extension {
             }).bind(this),
 
             getWifiEnabled: (function() {
-                return this._client.wireless_enabled;
+                return this._client !== null ? this._client.wireless_enabled : false;
             }).bind(this),
             setWifiEnabled: (function(arg) {
-                this._client.wireless_enabled = arg;
+                if (this._client !== null)
+                    this._client.wireless_enabled = arg;
             }).bind(this),
 
             getAirplaneModeEnabled: (function() {
-                return this._rfkillManager.airplaneMode;
+                // FIXME: why null is returned? Is the following assumption safe?
+                let ret = this._rfkillManager.airplaneMode;
+                return ret === null ? true : ret;
```

Comment:
On my testing at least the first call landing here got `null` from the `this._rfkillManager.airplaneMode` but I am not sure why.. And I don't know is it safe to assume the airplane mode be enabled when getting `null`; but on my machine when I boot the airplane mode seems to be enabled.